packages/color-scale-block/src/components/ColorPickerFlyout.tsx

```diff
@@ -0,0 +1,83 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { useState } from 'react';
+import { Color, ColorFormat, ColorPicker, Flyout, FlyoutPlacement } from '@frontify/fondue';
+import { ColorPickerFlyoutProps } from '../types';
+
+export const ColorPickerFlyout = ({
+    newIndex,
+    updateColor,
+    editedColor,
+    setEditedColor,
+    isColorPickerOpen,
+    setIsColorPickerOpen,
+    colors,
+}: ColorPickerFlyoutProps) => {
+    const [colorPickerFormat, setColorPickerFormat] = useState(ColorFormat.Hex);
+    const onCancel = () => {
```

Comment:
Rename `on` functions to `handle` so `onCancel` becomes `handleCancelClick` etc.